examples/semantic_segmentation/semantic_segmentation/workflow.py

```diff
@@ -37,12 +37,7 @@ class TestSample(Image):
 
 @dataclass(frozen=True)
 class GroundTruth(BaseGroundTruth):
```

Comment:
If you pass BaseGroundTruth to the workflow directly we can use `kolena.workflow.groundtruth` everywhere

Reply:
I tried that initially, but the base `GroundTruth` class has to be sub-classed: https://github.com/kolenaIO/kolena/blob/2e3417d1e8e690aaafcb0c7b1f75eb51c9f92af4/kolena/workflow/ground_truth.py#L97

Reply:
Does this not work with `kolena.workflow.GroundTruth`?
```
>>> from kolena.workflow import GroundTruth
>>> issubclass(GroundTruth, GroundTruth)
True
```